samples/subsys/video/capture_to_lvgl/README.rst

```diff
@@ -0,0 +1,82 @@
+.. zephyr:code-sample:: video-capture-to-lvgl
+   :name: Video capture to LVGL
+   :relevant-api: video_interface
+
+   This sample application demonstrates how to use the video API to retrieve video
+   frames from a capture device and display them on an LCD via the LVGL library.
+
+Description
+***********
+
+The application uses the :ref:`Video API <video_api>` to retrieve video frames from
+a video capture device, writes a frame count message to the console, and then sends
+the frame to an LCD display.
+
+Requirements
+************
+
+This sample requires a video capture device (e.g., a camera) and an LCD display.
```

Comment:
doesn't have to be *LCD* specifically, right?

```suggestion
This sample requires a supported :ref:`video capture device <video_api>` (e.g., a camera) and a :ref:`display <display_api>`.
```

Reply:
Changed